app/services/symphony_client.rb

```diff
@@ -220,7 +220,14 @@ def request(path, headers: {}, method: :get, **other)
         .headers(default_headers.merge(headers))
         .request(method, base_url + path, **other)
     end
-
+  # def request(path, headers: {}, method: :get, **other)
+  # logger = Logger.new(STDOUT)
+  # HTTP
+  #     .use(logging: { logger: logger },
```

Comment:
curious why you added then commented out logging? did not work as you expect or decided not to use?

Reply:
Because I don't always want it on, it's just here so I can swap it with the other one when i want more data